data/transition-sites/mod_sabre.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: mod_sabre
+whitehall_slug: ministry-of-defence
+homepage: https://www.gov.uk/government/organisations/ministry-of-defence
+host: www.sabre.mod.uk
+tna_timestamp: 20150806090301
+aliases:
+  - sabre.mod.uk
```

Comment:
We've standardised on no indentation for the YAML files in this repo (there isn't a check for that in the build but it would be nice to add one).